src/import-v1/c32helper.ts

```diff
@@ -0,0 +1,102 @@
+// The b58ToC32 function from https://github.com/blockstack/c32check, optimized for speed in NodeJS.
```

Comment:
is there a need for separate test cases for this?

Reply:
Tests added